### PR TITLE
numerous small nits after runthrough

### PR DESCRIPTION
**Pull request by xginn8**
Change-type: patch
Signed-off-by: Matthew McGinn <matthew@balena.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,7 +123,7 @@ on a specific path, and are stored on the host filesystem without any
 sort of diff mechanism (that is, any file hierarchy data written to the volume
 is always available in that hierarchy).
 
-For single service applications (ie. non-multicontainer applications without
+For single service applications (eg. non-multicontainer applications without
```
Comment (garethtdavies) [inline README.md]:
```suggestion
For single service applications (e.g. non-multicontainer applications without
```
We don't have a style guide so 🤷‍♂ but e.g. would be preferable if we had one :)

File: README.md [hunk 2]
```diff
@@ -654,7 +657,7 @@ the `frontend` is exposed to the host and beyond.
 
 To do this, we're going to remove host network access from the `frontend`
 service to allow external incoming network traffic into it, as well as ensure
-only incoming requests from the same bridge (ie. the `frontend` service) are
+only incoming requests from the same bridge (eg. the `frontend` service) are
```
Comment (garethtdavies) [inline README.md]:
```suggestion
only incoming requests from the same bridge (i.e. the `frontend` service) are
```
nit but I'd say i.e. is the right use there as we are referring to the actual `frontend` service here.

**Review by garethtdavies — COMMENTED** [review]
LGTM, a couple of minor nits which we probably simply need to standardize on.

**Review by garethtdavies — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (xginn8) [reply README.md]: Updated!

File: README.md (hunk 2 above)
Comment (xginn8) [reply README.md]: Yeah, fair enough. I kind of assumed they were interchangeable, but after closer examination `i.e.` is "in other words" and `e.g.` is "for example" :)

**Pull request merged**